test/e2e/steps/hooks.js

```diff
@@ -10,9 +11,18 @@ Before('@loginHook', function () {
   LoginPage.submitButton.click();
 });
 
+Before('@createList', function () {
+  DashboardPage.addListButton.moveTo();
+  DashboardPage.addListButton.click();
+  DashboardPage.newListButton.click();
```

Comment:
Is it necessary to do double click? 
Maybe a wait?

Reply:
Those are clicks for different elements.